resources/assets/js/app/collections/commentCollection.js

```diff
@@ -5,7 +5,7 @@ module.exports = BaseCollection.extend({
     model: model,
     url: '/comments',
     initialize: function (models, options) {
-        if (options.parentUrl) {
+        if (options && options.parentUrl) {
```

Comment:
implement this logic in BaseCollection, we have a lot of same initializers accross app